### ISSUE TITLE
Report List: Aggregate similar entries?

### ISSUE THREAD

**Issue by ghost** [idmc-labs/idmc-monitoring-platform-feedback#3]
Is it possible to somehow aggregate similar entries... many entries talk about the same event or even are even the same article but published in multiple sources... 

![screen shot 2018-03-05 at 10 59 25 am](https://user-images.githubusercontent.com/1164551/36993910-5046fb40-2064-11e8-9916-de4b6a8171c3.png)

**Comment by andreslizcano** [comment]
Yes, this makes the tool hard to use: sometimes I have the same articles dozens of times (in different publications).
